<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>selectPayments</class>
 <widget class="QWidget" name="selectPayments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Approve Payments</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="margin">
      <number>12</number>
     </property>
     <property name="spacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="3">
          <widget class="VendorGroup" name="_vendorgroup" native="true"/>
         </item>
         <item row="1" column="0">
          <widget class="XLabel" name="_selectDateLit">
           <property name="text">
            <string>Due Date:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="XComboBox" name="_selectDate">
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>On or Before</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Between</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QStackedWidget" name="_duedateStack">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="_allPage"/>
           <widget class="QWidget" name="_olderThanPage">
            <layout class="QGridLayout" name="_2">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="_3">
               <item>
                <widget class="DLineEdit" name="_onOrBeforeDate" native="true">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="_betweenPage">
            <layout class="QGridLayout" name="_4">
             <item row="0" column="0">
              <widget class="DLineEdit" name="_startDate" native="true"/>
             </item>
             <item row="0" column="1">
              <widget class="XLabel" name="_andLit">
               <property name="text">
                <string>and</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="DLineEdit" name="_endDate" native="true"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="_close">
           <property name="text">
            <string>&amp;Close</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_print">
           <property name="text">
            <string>&amp;Print</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="_openAPItemsLit">
            <property name="text">
             <string>Open Items:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="XTreeWidget" name="_apopen">
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="_select">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Approve...</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_selectLine">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Approve Line</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_clear">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Clear Approval</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Preferred</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="_selectDue">
                <property name="text">
                 <string>Approve all Due</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_selectDiscount">
                <property name="text">
                 <string>Approve all Discounts</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_clearAll">
                <property name="text">
                 <string>Clear Approvals</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_applyallcredits">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Apply all Credits</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>16</width>
                  <height>46</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="_bankaccntLit">
            <property name="text">
             <string>Use default Bank Account:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="XComboBox" name="_bankaccnt">
              <property name="allowNull" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::MinimumExpanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>DLineEdit</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VendorGroup</class>
   <extends>QWidget</extends>
   <header>vendorgroup.h</header>
  </customwidget>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XLabel</class>
   <extends>QLabel</extends>
   <header>xlabel.h</header>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_apopen</tabstop>
  <tabstop>_select</tabstop>
  <tabstop>_selectLine</tabstop>
  <tabstop>_clear</tabstop>
  <tabstop>_selectDue</tabstop>
  <tabstop>_selectDiscount</tabstop>
  <tabstop>_clearAll</tabstop>
  <tabstop>_bankaccnt</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_apopen</sender>
   <signal>valid(bool)</signal>
   <receiver>_clear</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>selectPayments</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_apopen</sender>
   <signal>valid(bool)</signal>
   <receiver>_selectLine</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_selectDate</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>_duedateStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_apopen</sender>
   <signal>valid(bool)</signal>
   <receiver>_select</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
